examples/scripts/sft_tldr.py

```diff
@@ -0,0 +1,190 @@
+# flake8: noqa
```

Comment:
I trained an SFT model before doing SRPO. I thought this example might be nice to have but I can remove if we do not think it's necessary.